### DISCUSSION TITLE
Price transparency 2024

### DISCUSSION THREAD

**Discussion by hghdinss** [CMSgov/hospital-price-transparency#130] [Q&A] [answered]
Hi, I am new to Charge Services and am trying to work on the price transparency. We are wondering if there were examples, in lay man terms, of what is needed for each column. We have CMS guidebooks to help with the rules, but without understanding exactly what is being needed in the columns, we are struggling to get this done. We have done lines 1-3 without any errors. We have added the description but are not sure if there can be 1 for each charge or does there need to be 1 line of same charge per payer/plan? Example: inpatient charge, is there a general charge or do we need one for OB, Med/Surg all separate according to payer/plan? I am not sure if any of this makes sense, but we need a lot of guidance in easy-to-understand terms. Is there a better place via the web we can go to find easy to read templates of information needed?

**Comment by RidenourK** [comment, marked as answer]
@hghdinss. Thank you for your question. 

Please see the [CSV Wide](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples/CSV/Wide%20Format%20Examples), [CSV Tall](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples/CSV/Tall%20Format%20Examples) or [JSON](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples/JSON) examples on this GitHub site, depending on the format your hospital has selected to use, for examples of how to indicate items and services in the MRF.
